presto-main/src/main/java/com/facebook/presto/server/PrestoSystemRequirements.java

```diff
@@ -36,16 +36,33 @@ private PrestoSystemRequirements() {}
 
     public static void verifyJvmRequirements()
     {
-        String specVersion = StandardSystemProperty.JAVA_SPECIFICATION_VERSION.value();
-        if ((specVersion == null) || (specVersion.compareTo("1.8") < 0)) {
-            failRequirement("Presto requires Java 1.8+ (found %s)", specVersion);
-        }
-
         String vendor = StandardSystemProperty.JAVA_VENDOR.value();
         if (!"Oracle Corporation".equals(vendor)) {
             failRequirement("Presto requires an Oracle or OpenJDK JVM (found %s)", vendor);
         }
 
+        String javaVersion = StandardSystemProperty.JAVA_VERSION.value();
+        if (javaVersion == null) {
+            failRequirement("Java version not defined");
+        }
+        String[] versionParts = javaVersion.split("_");
+        if (versionParts.length != 2) {
+            failRequirement("Java version has an unknown format: %s", javaVersion);
+        }
+        String majorVersion = versionParts[0];
+        int minorVersion;
+        try {
+            minorVersion = Integer.parseInt(versionParts[1]);
+        }
+        catch (Exception e) {
+            minorVersion = 0;
+        }
+
+        if ((majorVersion == null) || (majorVersion.compareTo("1.8.0") < 0) ||
+                (majorVersion.compareTo("1.8.0") == 0 && minorVersion < 60)) {
+            failRequirement("Presto requires Java 1.8.0_60+ (found %s_%d)", majorVersion, minorVersion);
```

Comment:
Probably better to give the "marketing name" and print the exact version string (for debugging when someone reports an error):
```java
failRequirement("Presto requires Java 8u60+ (found %s)", javaVersion);